File: python_ToSXs/python/06-thaphinh/86.py
height = 5

record = [str(j) for j in range(1, height + 1, 1)]
record = "".join(record) + "\n"
shape = record

for i in range(2, height):
    record = str(i) + " " * (height - 2) + str(i + height - 2 + 1)
    shape += record + "\n"

record = [str(j) for j in range(height, height * 2, 1)]
record = "".join(record)
shape += record

print(shape)

File: cpp_e9zMZ/cpp/06-thaphinh/86.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    int height = 5;
    string shape = createListNumber(1, height) + "\n";

    for (int i = 2; i < height; i++) {
        string left = to_string(i);
        string right = to_string(i + height - 1);
        string record = left + myRepeat(" ", height - 2) + right;
        shape += record + "\n";
    }

    shape += createListNumber(height, 2 * height - 1);
    cout << shape << "\n";
}